frontend/components/wallet/WalletDepositWithdrawBlock.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+    <div class="WalletDepositWithDrawBlock">
+        <TextBlock v-if="isExplanationsShown" title="Moving funds">
+            The DAI funds can be moved freely between VAT and your wallet without any extra conversion. You only need
+            to be aware that each transfer incurs transaction fees. Also, if you do not have DAI funds to deposit yet,
+            there are several ways to obtain it:
+            <ul class="list-disc list-inside">
+                <li>By borrowing DAI against a collateral in the oasis.app</li>
+                <li>
+                    By purchasing it on a decentralized exchange like uniswap.org (correct DAI token address used on
+                    the “mainnet” network is
+                    <FormatAddress type="address" :value="tokenAddressDAI" shorten />)
+                </li>
+            </ul>
+        </TextBlock>
+        <form class="flex flex-col gap-4" @submit.prevent="submit">
+            <RadioGroup
+                :disabled="isLoading"
+                :value="selectedMethod"
+                button-style="solid"
+                class="flex items-center w-full"
+                @change="handleMethodChange"
+            >
+                <RadioButton value="deposit" class="w-full text-center"> Deposit </RadioButton>
+                <RadioButton value="withdraw" class="w-full text-center"> Withdraw </RadioButton>
+            </RadioGroup>
+
+            <BidInput
+                v-if="selectedMethod === 'deposit'"
+                key="depositInput"
+                :amount-to-bid.sync="depositAmount"
+                :disabled="isLoading"
+                :total-price="maxDeposit"
+                :minimum-deposit-dai="minimumDaiAmount"
+            />
+            <BidInput
+                v-else
+                key="withdrawInput"
+                :amount-to-bid.sync="withDrawAmount"
+                :disabled="isLoading"
+                :total-price="maxWithdraw"
+                :minimum-deposit-dai="minimumDaiAmount"
+            />
+
+            <BaseButton type="primary" class="capitalize" :disabled="isLoading || !canSubmit" html-type="submit">
```

Comment:
So there is should be two types of being disabled:

- When the form was submitted and `isLoading` (therefore please pass `isLoading` to the button, so it's animated)
- When the form is disabled, eg till the depositing is allowed. Since depositing and withdrawing require different auth steps, we may want to disable them separately: let's add two new props `isDepositingAllowed` and `isWithdrawingAllowed` and disable them separately